src/main/kotlin/com/nextstep/lotto/domain/Lotto.kt

```diff
@@ -0,0 +1,17 @@
+package com.nextstep.lotto.domain
+
+data class Lotto(val lottoNumbers: List<LottoNumber>) {
+    init {
+        validate(lottoNumbers)
+    }
+
+    private fun validate(lottoNumbers: List<LottoNumber>) {
+        if (lottoNumbers.size != NUMBER_OF_LOTTO_NUMBER) {
+            throw IllegalArgumentException("로또는 6개의 숫자로 이루어져 있습니다.")
+        }
```

Comment:
require를 한번 활용해보세요.

```suggestion
require(lottoNumbers.size == NUMBER_OF_LOTTO_NUMBER) { "로또는 6개의 숫자로 이루어져 있습니다. "}
```

위와 같이 정의하면 내부에서 `IllegalArgumentException` 를 던지도록 구현되어 있어요~